test/functional/p2p_invalid_messages.py

```diff
@@ -83,9 +83,8 @@ def test_size(self):
 
     def test_msgtype(self):
         conn = self.nodes[0].add_p2p_connection(P2PDataStore())
-        with self.nodes[0].assert_debug_log(['PROCESSMESSAGE: ERRORS IN HEADER']):
+        with self.nodes[0].assert_debug_log(['HEADER ERROR - COMMAND']):
```

Comment:
3a9d277
```suggestion
        with self.nodes[0].assert_debug_log([
                'HEADER ERROR - COMMAND (bad, 2 bytes), peer=',
                'disconnecting peer=',
        ]):
```

Reply:
I'm going to leave this one.  The reason the command is wrong is not because it's an unrecognized command, it's because it isn't validly constructed.  It ends up being `bad\x00sg\x00\x00\x00\x00\x00\x00` which doesn't pass the "once you have a `\x00` all remaining chars must be `\x00`" test.  One day it might be nice to change all of these tests so that they send recognized commands, and then check for that separately.